doc/source/data/overview.rst

```diff
@@ -15,53 +15,41 @@ Ray Data is a scalable data processing library for ML workloads, particularly su
 
-.. note::
-
-    Ray Data doesn't have a SQL interface and isn't meant as a replacement for generic
-    ETL pipelines like Spark.
-
 Why choose Ray Data?
 --------------------
 
 .. dropdown:: Faster and cheaper for modern deep learning applications
 
-    Ray Data is designed for deep learning applications that involve both CPU preprocessing and GPU inference. Through its powerful streaming :ref:`Dataset <dataset_concept>` primitive, Ray Data streams working data from CPU preprocessing tasks to GPU inferencing or training tasks, allowing you to utilize both sets of resources concurrently.
+    Ray Data is designed for deep learning applications that involve both CPU preprocessing and GPU inference. Through its powerful streaming execution, Ray Data streams working data from CPU preprocessing tasks to GPU inferencing or training tasks, allowing you to utilize both sets of resources concurrently.
 
     By using Ray Data, your GPUs are no longer idle during CPU computation, reducing overall cost of the batch inference job.
 
 .. dropdown:: Cloud, framework, and data format agnostic
 
     Ray Data has no restrictions on cloud provider, ML framework, or data format.
 
-    Through the :ref:`Ray cluster launcher <cluster-index>`, you can start a Ray cluster on AWS, GCP, or Azure clouds. You can use any ML framework of your choice, including PyTorch, HuggingFace, or Tensorflow. Ray Data also does not require a particular file format, and supports a :ref:`wide variety of formats <loading_data>` including CSV, Parquet, and raw images.
+    You can start a Ray cluster on AWS, GCP, or Azure clouds. You can use any ML framework of your choice, including PyTorch, HuggingFace, or Tensorflow. Ray Data also does not require a particular file format, and supports a :ref:`wide variety of formats <loading_data>` including Parquet, images, JSON, text, CSV, etc.
+
+.. dropdown:: Out of the box scaling on heterogeneous cluster
 
-.. dropdown:: Out of the box scaling
+    Ray Data is built on Ray, so it easily scales on a heterogeneous cluster, which has different types of CPU and GPU machines. Code that works on one machine also runs on a large cluster without any changes.
 
-    Ray Data is built on Ray, so it easily scales to many machines. Code that works on one machine also runs on a large cluster without any changes.
+    Ray Data can easily scale to hundreds of nodes to process hundreds of TB of data.
 
-.. dropdown:: Python first
+.. dropdown:: Unified API and backend for batch inference and ML training
 
-    With Ray Data, you can express your inference job directly in Python instead of
-    YAML or other formats, allowing for faster iterations, easier debugging, and a native developer experience.
+    With Ray Data, you can express batch inference and ML training job directly under the same Ray Dataset API.
```

Comment:
I know this is also in the draft for the blog post, but IMO "Python first" is more compelling than "Unified API and backend for batch inference and ML training."

Don't need to block on this since we can always revise later.